tools/node-modules

```diff
@@ -0,0 +1,173 @@
+#!/bin/sh
+
+set -eu
+
+CACHE_DIR="${XDG_CACHE_HOME-${HOME}/.cache}/cockpit-dev/node-cache.git"
+
+GITHUB_BASE="${GITHUB_BASE:-cockpit-project/cockpit}"
+GITHUB_REPOSITORY="${GITHUB_BASE%/*}/node-cache"
+CLONE_REMOTE="https://github.com/${GITHUB_REPOSITORY}"
+PUSH_REMOTE="git@github.com:${GITHUB_REPOSITORY}"
+
+v0() {
+    printf "  %-8s %s\n" "$1" "$2"
```

Comment:
Please move the `${V}` check here and call this `log()` or `message()` or something such, as suggested in the earlier review round.

Reply:
I wanted to keep the `${V}` check out of here to prevent seeing the info output in `set +x` mode in case `V` != `0`.  But maybe it's not so awful...